site/guide.md

```diff
@@ -428,7 +428,7 @@ import cats.effect.IO
 Stream(1,2,3).merge(Stream.eval(IO { Thread.sleep(200); 4 })).compile.toVector.unsafeRunSync()
```

Comment:
Ugh, this guide is old. There are many things wrong with it. This `Thread.sleep` should be rewritten as `IO.sleep(...).as(4)` or something.